components/cosmetic_filters/resources/data/element_picker.html

```diff
@@ -101,7 +101,7 @@
     }
   }
 
-  .minimized:not(.desktop) {
+  :host-context(#brave-element-picker.minimized) #main-section:not(.desktop) {
```

Comment:
nit: we have a lot of these selectors. I think its probably worth using CSS nesting here.
https://developer.mozilla.org/en-US/docs/Web/CSS/Nesting_selector

```css
  :host-context(#brave-element-picker.minimized) {
    &  #main-section:not(.desktop) {
      transition: transform 0.3s ease;
      transform: translateY(calc(100% - 28px));
    }

    & .desktop .drag-header, & .desktop .drag-indicator {
      ...
    }

    ...
  }
```